services/php-apache/Dockerfile

```diff
@@ -3,10 +3,11 @@ ENV COMPOSER_NO_INTERACTION=1 \
     COMPOSER_ALLOW_SUPERUSER=1 \
     DOCROOT=/var/www/html
 
-RUN apt-get install -y libfreetype6-dev libjpeg62-turbo-dev libmcrypt-dev libpng-dev mysql-client && \
+RUN apt-get install -y libfreetype6-dev libjpeg62-turbo-dev libmcrypt-dev libpng-dev && \
+    (apt-get install -y mysql-client || apt-get install -y mariadb-client) && \
```

Comment:
Interesting, why the or here?

Reply:
buster doesn't have a `mysql-client` package, only `mariadb-client`